### PR TITLE
Fix Wilds Witch NEI drop page + modpack loading speed

### PR DESCRIPTION
**Pull request by kuba6000**

### CODE REVIEW

File: src/main/java/toast/specialMobs/entity/witch/Entity_SpecialWitch.java [hunk 1]
```diff
@@ -394,6 +403,13 @@ protected void dropFewItems(boolean hit, int looting) {
         }
     }
 
+    @Optional.Method(modid = "mobsinfo")
+    @Override
+    public void provideDropsInformation(@Nonnull ArrayList<MobDrop> drops) {
+        IMobInfoProvider.provideSuperVanillaDrops(drops, EntityWitch.class);
+        // TODO: what to do with drinking potion ????
```
Comment (OneEyeMaker) [inline src/main/java/toast/specialMobs/entity/witch/Entity_SpecialWitch.java]: I think there should be added all potions mentioned in `tryDrinkPotion` method with chance of 15% (as stated in `dropFewItems`). At least what I can say looking at the code. The better approach would probably be to verify these values by play testing.

File: src/main/java/toast/specialMobs/entity/spider/EntityMotherSpider.java [hunk 2]
```diff
@@ -47,11 +55,10 @@ public void adjustTypeAttributes() {
     /// Called when this entity is killed.
     @Override
     protected void dropFewItems(boolean hit, int looting) {
+        // ALL CHANGES IN HERE MUST BE ALSO MADE IN provideDropsInformation
         super.dropFewItems(hit, looting);
         if (hit && (this.rand.nextInt(3) == 0 || this.rand.nextInt(1 + looting) > 0)) {
-            this.entityDropItem(
-                    new ItemStack(Items.spawn_egg, 1, EntityList.getEntityID(new EntitySpider(this.worldObj))),
-                    0.0F);
+            this.entityDropItem(new ItemStack(Items.spawn_egg, 1, 52 /* EntitySpider */), 0.0F);
```
Comment (OneEyeMaker) [inline src/main/java/toast/specialMobs/entity/spider/EntityMotherSpider.java]: I understand that creating a new entity whenever you need to just retrieve its id is wasteful. But is there any other way to get that *magic* number?

Comment (kuba6000) [reply src/main/java/toast/specialMobs/entity/spider/EntityMotherSpider.java]: I changed it to `net.minecraft.entity.EntityList#classToIDMapping`

File: src/main/java/toast/specialMobs/entity/witch/Entity_SpecialWitch.java (hunk 1 above)
Comment (kuba6000) [reply src/main/java/toast/specialMobs/entity/witch/Entity_SpecialWitch.java]: Will look at it later, it has to be fixed on normal witch drops as well

**Review by OneEyeMaker — APPROVED** [review]

**Pull request merged**